### PR TITLE
Add New flipper for Real User Monitoring of Disability Benefits Frontends

### PR DESCRIPTION
**Pull request by tfink419**
## Summary
- Simple flipper addition for frontend ticket for adding Datadog's Real User Monitoring to some pages

## Related issue(s)
- https://github.com/department-of-veterans-affairs/vets-website/pull/31820

### CODE REVIEW

File: config/features.yml [hunk 1]
```diff
@@ -1135,7 +1135,7 @@ features:
     description: Enables VA Form 5655 (Financial Status Report)
     enable_in_development: true
   show_financial_status_report_wizard:
-    actor_type: user
+    actor_type: use
```
Comment (micahaspyr) [inline config/features.yml:1138]: This should be user

**Review by micahaspyr — CHANGES_REQUESTED** [review]
Just the small change from 'use' to 'user' otherwise looks good

File: config/features.yml (hunk 1 above)
Comment (tfink419) [reply config/features.yml:1138]: Oh good catch

**Review by micahaspyr — APPROVED** [review]
LGTM

**Review by rjohnson2011 — APPROVED** [review]

**Pull request merged**